pkg/operator/observe_config.go

```diff
@@ -99,6 +103,25 @@ func (c ConfigObserver) sync() error {
 		unstructured.SetNestedStringSlice(observedConfig, etcdURLs, "storageConfig", "urls")
 	}
 
+	internalRegistryHostName := ""
+	imageConfigClient, err := configv1.NewForConfig(c.clientConfig)
+	if err != nil {
+		return err
+	}
+	configImageList, err := imageConfigClient.Images().List(metav1.ListOptions{})
+	if err != nil {
+		return err
+	}
+	for _, imageConfig := range configImageList.Items {
+		if len(imageConfig.Status.InternalRegistryHostname) > 0 {
+			internalRegistryHostName = imageConfig.Status.InternalRegistryHostname
```

Comment:
@deads2k or @bparees Am I looking for a specific image config resource? Will there ever only be just one object in the openshift-image-registry namespace?

Reply:
and it'll be cluster scoped, not in the openshift-image-registry namespace.